go-controller/pkg/node/gateway_iptables.go

```diff
@@ -78,20 +79,35 @@ func delIptRules(rules []iptRule) error {
 }
 
 func getGatewayInitRules(chain string, proto iptables.Protocol) []iptRule {
-	return []iptRule{
-		{
-			table:    "nat",
-			chain:    "PREROUTING",
-			args:     []string{"-j", chain},
-			protocol: proto,
-		},
-		{
+	iptRules := []iptRule{}
+	if chain == iptableITPChain {
```

Comment:
nit: this is a little awkward to me because its conflating the chain with tables, and even if the chain isn't OUTPUT still returns nat OUTPUT chain, but not a big deal

Reply:
yes,, I understand the confusion. Its called from nat OUTPUT for the backend being host-networked pods case and from mangle OUTPUT for the backend being ovn pods case.